scene/main/node.h

```diff
@@ -320,7 +321,7 @@ class Node : public Object {
 	GDVIRTUAL0(_enter_tree)
 	GDVIRTUAL0(_exit_tree)
 	GDVIRTUAL0(_ready)
-	GDVIRTUAL0RC(Vector<String>, _get_configuration_warnings)
+	GDVIRTUAL0RC(Array, _get_configuration_warnings)
```

Comment:
This breaks binary compatibility for GDExtension. Any GDExtension that implements `_get_configuration_warnings()` from before this change will probably segfault. Does anyone know if there's an issue discussing this yet? If not, I can make one

Reply:
I just created issue https://github.com/godotengine/godot/issues/88379